app/experimenter/nimbus-ui/src/components/PageResults/TableMetricSecondary/index.tsx

```diff
@@ -56,11 +61,33 @@ const TableMetricSecondary = ({
     : METRIC_TYPE.USER_SELECTED_SECONDARY;
 
   const overallResults = results?.overall!;
+  const probeSetName =
+    results.metadata?.metrics[probeSetSlug]?.friendly_name ||
+    probeSetDefaultName;
+  const probeSetDescription =
+    results.metadata?.metrics[probeSetSlug]?.description || undefined;
+
+  const popover = (
+    <Popover id="metadata">
+      <Popover.Content>
+        <ReactMarkdown source={probeSetDescription!} />
+      </Popover.Content>
+    </Popover>
+  );
 
   return (
     <div data-testid="table-metric-secondary" className="mb-5">
       <h2 className="h5 mb-3" id={probeSetSlug}>
-        <div>{probeSetName}</div>
+        <div>
+          <div className="d-inline-block">
+            {probeSetName}{" "}
+            {probeSetDescription && (
+              <OverlayTrigger overlay={popover} trigger="click" rootClose>
```

Comment:
I might be missing something since I'm doing a quick end-of-day review, but I checked out the docs for this to get an idea of what I should be seeing, and I'm wondering why we wouldn't use the `react-tooltip` for consistency but with `event="click"` if we specifically want it to show onclick (if I'm [reading their docs](https://www.npmjs.com/package/react-tooltip#options) correctly)? Maybe we have to or want to use `ReactMarkdown` and it's difficult to get that working with a tooltip?

Also, I think Storybook might need some love, I think we should be seeing that pop up here but when I hover/click the info icon I'm not seeing anything.

<img width="302" alt="image" src="https://user-images.githubusercontent.com/13018240/107711890-06ca2900-6c8e-11eb-974a-f0ea94fd1b38.png">